messagebird/client.py

```diff
@@ -356,5 +357,8 @@ def voice_recording_download(self, call_id, leg_id, recording_id):
         recording_file = self.request_store_as_file(VOICE_API_ROOT + recording_file, recording_id + '.wav')
         return VOICE_API_ROOT + recording_file
 
+    def call_flow(self, id):
+        return CallFlow().load(self.request('call-flows/' + str(id), 'GET', None))
```

Comment:
Don't think this will work, needs to have the voice type as to set the proper domain/url to `https://voice.messagebird.com/call-flows/`.
```suggestion
        return CallFlow().load(self.request('call-flows/' + str(id), 'GET', None, VOICE_TYPE))
```